src/providers/postgres/qgspostgresconn.h

```diff
@@ -183,6 +183,12 @@ class QgsPostgresConn : public QObject
     Q_OBJECT
 
   public:
+    /// @param shared allow using a shared connection. Should never be
```

Comment:
Please deprecate it if it's the right thing to do. Deprecating does not hurt but helps to make future proof code as early as possible.